Please check this test failure:

```
Error: 8,010 [ERROR] Failures: 
Error: 8,010 [ERROR]   MetadataSharingApiTest.settingPrivilegesToNoneCreatesPrivilegesHistoryStatusEntry:389 expected:<1> but was:<0>
07:59:08,010 [INFO] 
Error: 8,010 [ERROR] Tests run: 278, Failures: 1, Errors: 0, Skipped: 5
```

--- a/services/pom.xml
+++ b/services/pom.xml
@@ -137,6 +137,12 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>gn-listeners</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
 
     <dependency>
       <groupId>com.google.code.gson</groupId>

--- a/services/src/test/resources/services-web-test-context.xml
+++ b/services/src/test/resources/services-web-test-context.xml
@@ -2,9 +2,11 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:mvc="http://www.springframework.org/schema/mvc"
+       xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
-       http://www.springframework.org/schema/mvc  http://www.springframework.org/schema/mvc/spring-mvc.xsd">
+       http://www.springframework.org/schema/mvc  http://www.springframework.org/schema/mvc/spring-mvc.xsd http://www.springframework.org/schema/context https://www.springframework.org/schema/context/spring-context.xsd">
 
+  <context:component-scan base-package="org.fao.geonet.listener"/>
 
   <bean id="cnManager"
         class="org.springframework.web.accept.ContentNegotiationManagerFactoryBean">

